schema/schemaExaminerSchedule.json

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
This schema needs updating with the latest from the journal service PR. It will eventually be in it's own repo + NPM package.